my_game.py

```diff
@@ -7,16 +7,33 @@
 """
 
 import arcade
+import arcade.gui
 import math
 import random
 import tomli
-import arcade.gui
+import tomli_w
+import pathlib
 
 from game_sprites import Star
 
 # load the config file as a dict
-with open('my_game.toml', 'rb') as fp:
-    CONFIG = tomli.load(fp)
+def load_my_game_to_CONFIG():
```

Comment:
This function should _return tomli.load(fp)_. Having the function depend on a global variable is bad design. Functions should not have external dependencies. The function should maybe be called _load_toml(file_to_load)_. 

Then use it like this:
```
CONFIG = load_toml("some_file.toml")
```